docs/guides/query-kyc-state-on-c-chain.md

```diff
@@ -0,0 +1,164 @@
+<figcaption align="center"><b>Fig.6:</b> Verify network is connected</figcaption>
+</figure>
+
+## Downloading the CaminoAdmin.abi file
+
+We can either compile the contract or use the precompiled ABI file. In this guide, we will use the ABI file.
+
+You can download the ABI file from Chain4Travel's GitHub repository:
+[chain4travel/caminoethvm - CaminoAdmin.abi](https://github.com/chain4travel/caminoethvm/blob/chain4travel/contracts/build_contracts/admin/abi/CaminoAdmin.abi)
+
+- Rigth click on "Raw" and select "Save as..." to save it to your local computer.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/6-abi-save-link-as.png"/>
+<figcaption align="center"><b>Fig.7:</b> Save "CaminoAdmin.abi" file to local computer</figcaption>
+</figure>
+
+## Uploading the ABI file to the Remix IDE
+
+Follow these steps to upload the ABI file to the Remix IDE:
+
+- Switch to **File Explorer** on the right sidebar and click the upload button.
+- Select the file you downloaded from your local computer and upload it.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/7-load-local-file.png"/>
+<figcaption align="center"><b>Fig.8:</b> Switch to "File Explorer" and click "Load a local file into current workspace"</figcaption>
+</figure>
+
+- Select the `CaminoAdmin.abi` file.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/8-select-the-loaded-file.png"/>
+<figcaption align="center"><b>Fig.9:</b> Select the loaded ABI file</figcaption>
+</figure>
+
+## Getting the CaminoAdmin Contract Address
+
+**Camino Admin Address**: `0x010000000000000000000000000000000000000a`
+
+You can also find the Camino contract addresses on [Camino Smart Contracts](/developer/references/camino-contracts.md) page under `Developer` > `References`.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/9-get-the-contract-address.png"/>
+<figcaption align="center"><b>Fig.10:</b> Get the CaminoAdmin contract address</figcaption>
+</figure>
+
+## Interacting with the Address
+
+- Paste the address into the **At Address** field and click on the button.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/10-paste-address-and-click-at-address.png"/>
+<figcaption align="center"><b>Fig.11:</b> Paste the contract address into the "At Address" field and click "At Address"</figcaption>
+</figure>
+
+- Confirm the interaction
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/11-confirm-the-popup.png"/>
+<figcaption align="center"><b>Fig.12:</b> Confirm the dialog popup</figcaption>
+</figure>
+
+## Making the Query
+
+Follow the steps below to make the query:
+
+- Click the arrow to open the contracts methods list.
+
+<figure>
+<img class="zoom" src="/img/query-kyc-state-c-chain/12-open-dropdown-abi.png"/>
+<figcaption align="center"><b>Fig.13:</b> Click the arrow to open the dropdown</figcaption>
+</figure>
+
+- Paste the address you want to query KYC state into the field and click **`getKycState`** button.
```

Comment:
I think it might be important here to specify that it must be an address that starts with "0x" and needs to be the c-chain address, as it cannot even be the hashed public address that resolves all subnets I believe.

Reply:
Of course, this is a documentation for the C-Chain. Other chains have native API methods to query an address state (KYC etc). I'll add a note about this.